digital-deck-ui/src/components/MainMenu.vue

```diff
@@ -1,15 +1,57 @@
 <template>
   <div>
     <div v-if="!isLoaded" class="main-menu">
-      <h1>Digital Deck</h1>
+      <h1 class="heading">Digital Deck</h1>
+      <button class="button" v-on:click="toggleForm1()">Create</button>
+      <button class="button" v-on:click="toggleForm2()">Join</button>
       <br />
-      <span>Decks</span><br />
-      <input v-model.number="deckSelected" type="number" min="1" oninput="validity.valid||(value='');"/><br />
-      <span>Cards on hand</span><br />
-      <input v-model.number="cardsPerPlayer" type="number"  min="1" oninput="validity.valid||(value='');"/><br />
-      <span>Cards on table</span><br />
-      <input v-model.number="cardsOnTable" type="number"  min="0" oninput="validity.valid||(value='');"/><br /><br />
-      <button @click="submitForm" style="cursor: pointer;">Start</button>
+      <div v-if="form1" class="input-form">
+        <form>
+          <div class="number-box">
+            <input
+              v-model.number="deckSelected"
+              type="number"
+              min="1"
+              oninput="validity.valid||(value='');"
+              required=""
+            />
+            <label>Number of Decks</label>
+          </div>
+
+          <div class="number-box">
+            <input
+              v-model.number="cardsPerPlayer"
+              type="number"
+              min="1"
+              oninput="validity.valid||(value='');"
+              required=""
+            />
+            <label>Cards on hand</label>
+          </div>
+
+          <div class="number-box">
+            <input
+              v-model.number="cardsOnTable"
+              type="number"
+              min="1"
```

Comment:
The `min` value for `cardsPerPlayer` and `cardsOnTable` here should be 0. You can also make it so that the default value for `deckSelected` is 1 if you want.